tests/test_losses.py

```diff
@@ -4,12 +4,13 @@
 
 import unittest
```

Comment:
There don't appear to be any tests that apply label smoothing on actual instances of a loss class. Need to add that

Reply:
9480589  😇